examples/python/FullScriptSupport_Pandas/README.md

```diff
@@ -0,0 +1,45 @@
+
+## Content
+* [Implementation](#implementation)
+    * [Parameters sent from Qlik](#parameters-sent-from-qlik)
+    * [TableDescription](#tabledescription)
+    * [Result](#result)
+* [Qlik Documents](#qlik-documents)
+* [Run the Example!](#run-the-example)
+
+## Implementation
+We aim for a well documented code for you to easily be able to follow along. If something is unclear, please let us know so that we can update and improve our documentation. In this file we will guide you through a few specific key points in the implementation, that is worth clarifying.
+
+### Parameters sent from Qlik
+The parameters sent from Qlik are now stored in a `pandas.DataFrame` object called `q`. The names of the parameters, and hence the column names of `q`, are set to the names sent from Qlik in the _ScriptRequestHeader_. For instance if you send a parameter called `Foo` in Qlik, you will reach the parameter by writing `q.Foo` or `q["Foo"]` in the script.
+
+If the parameter is of type _Dual_ the plugin will create two additional columns in the `q` data frame, with the string and numerical representation. The column names will have the base as the parameter name but will end with '_str' and '_num' respectively. For example, a parameter called `Bar` with datatype _Dual_ will result in three columns in `q`: `Bar`, `Bar_str` and `Bar_num`. `Bar` will contain strings and numerics, `Bar_str` will contain only strings and `Bar_num` only numerics.
+
+### TableDescription
+In the load script, when using the `Load ... Extension ...` syntax you can create the `TableDescription` message within the script. This can be useful if you for instance want to name, set tags, or change datatype of the fields you are sending back to Qlik. Read more about what metadata can be included in the `TableDescription` in in the [SSE_Protocol.md](../../../docs/SSE_Protocol.md).
+
+An instance of the `TableDescription` message is available from the script by the name `table`. To that instance you can add metadata according to the protocol. A few simple examples:
+
+- `table.name = "Table1"` sets the table name to be _Table1_
+- `table.fields.add(name="firstField", dataType=1, tags=["tag1", "tag2"])` adds a _numeric_ field called _firstField_ with the tags _tag1_ and _tag2_.
+
+Note that if a `TableDescription` is sent, the number of fields in the message must match the number of fields of data sent back to Qlik.
+
+### Result
+With the change to using the `exec` method to evaluate the script, there are some changes regarding what's possible to write in the script. See the Python documentation of `exec` [here](https://docs.python.org/3/library/functions.html#exec). One of them being that the `exec` method does not return anything. We must therefore set the result to a specific variable which we have chosen to call `qResult`. If nothing was set to the variable no data will be returned to Qlik. Note that `qResult` is not required to be a pandas data frame.  
```

Comment:
One change is that the `exec` method does not return anything. We must therefore set the result to a specific variable, which we have chosen to call `qResult`. If nothing was set to the variable, no data will be returned to Qlik. Note that `qResult` is not required to be a Pandas data frame.